### ISSUE TITLE
Can the library auto calibrate magnetic sensor?

### ISSUE THREAD

**Issue by steprun** [xioTechnologies/Fusion#25]
Can the library auto calibrate magnetic sensor?

**Comment by xioTechnologies** [comment]
No.  The algorithm assumes that magnetometer measurements are already calibrated.

**Comment by xioTechnologies** [comment]
[This application note](https://www.nxp.com/docs/en/application-note/AN4246.pdf) describes a solution and includes a worked example on page 15.

**Comment by shads73** [comment]
The link does not open for me, can you provide an up-to-date link to an example of calibration?

**Comment by xioTechnologies** [comment]
I do not have an issue with the link.  I suggest you try opening it on a different device or by using a different connection.

**Comment by steprun** [comment]
To search AN4246 at NXP web site

**Comment by shads73** [comment]
Thanks!
Some miracles, now I tried again - everything opens ...
Probably in Russia now such miracles will periodically occur)
